### PR TITLE
update minio-go and crypto dependency

### PR DESCRIPTION
**Pull request by SherinV**
Signed-off-by: Sherin Varughese <shvarugh@redhat.com>

**Related Issue:**  open-cluster-management/backlog#18796 and https://github.com/stolostron/backlog/issues/18832

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,13 +9,12 @@ require (
...
 	github.com/kennygrant/sanitize v1.2.4
-	github.com/open-cluster-management/governance-policy-propagator v0.0.0-20211012174109-95c3b77cce09
-	github.com/open-cluster-management/multicloud-operators-channel v1.0.1-0.20200930214554-fa55cf642642
-	github.com/open-cluster-management/multicloud-operators-deployable v0.0.0-20200925154205-fc4ec3e30a4d
-	github.com/open-cluster-management/multicloud-operators-placementrule v1.2.4-0-20210816-699e5
-	github.com/open-cluster-management/multicloud-operators-subscription v1.0.0-2020-05-12-21-17-19.0.20201009005738-cbe273a045ab
-	github.com/open-cluster-management/multicloud-operators-subscription-release v1.0.1-2020-06-08-14-28-27.0.20200819124024-818f01d780ff //Use 2.0 when available
+	github.com/open-cluster-management/multicloud-operators-channel v1.2.5-0-20211122-79abb30
+	github.com/open-cluster-management/multicloud-operators-deployable v1.2.4-0-20211122-24ac963
+	github.com/open-cluster-management/multicloud-operators-placementrule v1.2.4-0-20211122-be034
```
Comment (SherinV) [inline go.mod:14]: These repos haven't moved to stolostron yet. https://github.com/stolostron/multicloud-operators-placementrule/blob/be034e3a07533e04e4f59ba75e8b12804385db70/go.mod#L1

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**